sdk/eventhub/eventhubs-checkpointstore-table/test/tables-checkpointstore.spec.ts

```diff
@@ -0,0 +1,526 @@
+        }
+      });
+    });
+
+    describe("claimOwnership", function() {
+      it("claimOwnership call should succeed, if it has been called for the first time", async function(): Promise<
+        void
+      > {
+        const checkpointStore = new TableCheckpointStore(client);
+        const listOwnership = await checkpointStore.listOwnership(
+          "testNamespace.servicebus.windows.net",
+          "testEventHub",
+          "testConsumerGroup"
+        );
+        should.equal(listOwnership.length, 0);
+
+        const partitionOwnership: PartitionOwnership = {
+          ownerId: "Id1",
+          partitionId: "0",
+          fullyQualifiedNamespace: "testNamespace.servicebus.windows.net",
+          consumerGroup: "testConsumerGroup",
+          eventHubName: "testEventHub"
+        };
+        const partitionOwnershipArray = await checkpointStore.claimOwnership([partitionOwnership]);
+        should.equal(partitionOwnershipArray.length, 1);
+
+        const ownershipList = await checkpointStore.listOwnership(
+          "testNamespace.servicebus.windows.net",
+          "testEventHub",
+          "testConsumerGroup"
+        );
+
+        should.equal(ownershipList.length, 1, "Unexpected number of ownerships in list.");
+        should.equal(
+          ownershipList[0].ownerId,
+          "Id1",
+          "The 1st ownership item has the wrong ownerId."
+        );
+        should.equal(
+          ownershipList[0].consumerGroup,
+          "testConsumerGroup",
+          "The 1st ownership item has the wrong consumerGroup."
+        );
+        should.equal(
+          ownershipList[0].fullyQualifiedNamespace,
+          "testNamespace.servicebus.windows.net",
+          "The 1st fullyQualifiedNamespace item has the wrong fullyQualifiedNamespace."
+        );
+        should.equal(
+          ownershipList[0].eventHubName,
+          "testEventHub",
+          "The 1st ownership item has the wrong eventHubName."
+        );
+        should.equal(
+          ownershipList[0].partitionId,
+          "0",
+          "The 1st ownership item has the wrong partitionId."
+        );
+        should.exist(ownershipList[0].lastModifiedTimeInMs, "lastModifiedTimeInMs should exist.");
+        should.exist(ownershipList[0].etag, "etag should exist.");
+        debug(
+          `LastModifiedTime: ${ownershipList[0].lastModifiedTimeInMs!}, ETag: ${
+            ownershipList[0].etag
+          }`
+        );
+      });
+    });
+  });
+
+  describe("Runs tests on a populated table", function() {
+    beforeEach("creating table", async () => {
+      tableName = `table${new Date().getTime()}`;
+      client = new TableClient(
+        `https://${service.storageAccountName}.table.core.windows.net`,
+        tableName,
+        credential
+      );
+      await serviceClient.createTable(tableName);
+      const namespace = "blue.servicebus.windows.net";
+      const eventHubName = "blueHub";
+      const consumerConst = "$default";
```

Comment:
Minor: If you moved these to be above your `beforeEach`, you can reference them in your tests as well, rather than needing to hard-code (and possibly typo!) them :smile:

Reply:
Also, `consumerConst` could be renamed to `consumerGroup`.

Reply:
great i just made these updates